src/style.css

```diff
@@ -0,0 +1,141 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --background-color-blue: #00acdc;
+  --background-color-white: #fff;
+  --dark-color-text: #060b35;
+  font-family: "Roboto", sans-serif;
+}
+
+h6,
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background-color: var(--background-color-white);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__mac-image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__title {
+  margin-top: 40px;
+  display: block;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--dark-color-text);
+  line-height: 18px;
+  font-style: normal;
```

Comment:
If we have a value `normal`, then it does not need to be prescribed, it is there by default. 
Fix it everywhere
```suggestion
```